nx-X11/programs/Xserver/dix/devices.c

```diff
@@ -63,7 +63,7 @@ SOFTWARE.
 #include "site.h"
 #define	XKB_IN_SERVER
 #ifdef XKB
-#include <nx-X11/extensions/XKBsrv.h>
+#include <xkbsrv.h>
```

Comment:
is it correct to have no captial letters in this filename? XKBsrv.h vs. xkbsrv.h?